test/test.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const hexValues = require('../index.js')
+const expect = require('chai').expect
+const Buffer = require('buffer').Buffer
+const fs = require('fs')
+
+describe('fs module', function(){
+  describe('hexVal', function(){
+    it('The first document hex value should be first in the array', done => {
+      expect(hexValues.firstDoc[0]).to.deep.equal(hexValues.hexOrder[0])
```

Comment:
You also generally want to mock data for your test that is specifically only used for this test file.